assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -38,12 +38,20 @@ const { useSelect } = Data;
 const DashboardDetailsWidgetKeywordsTable = () => {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	let referenceSiteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	if ( referenceSiteURL.endsWith( '/' ) ) {
```

Comment:
This is likely where the E2E tests are failing: note that https://github.com/google/site-kit-wp/blob/50ffe5232a0046632f78bf50f88a0ee615eee5d2/assets/js/googlesitekit/datastore/site/info.js#L323 can return `undefined`, which means this can possibly execute `undefined.endsWith( '/' )`.

That's the error I'm seeing in Travis: https://travis-ci.com/github/google/site-kit-wp/jobs/404472155#L1773

```suggestion
	if ( referenceSiteURL && referenceSiteURL.endsWith( '/' ) ) {
```

would do the trick 😄

Reply:
@tofumatt I actually tried that but the build still failed. So I have reverted that commits a few hrs back.
With the logic you proposed, I saw this [build error](https://travis-ci.com/github/google/site-kit-wp/jobs/403824791#L1773)

As I understand, I have to load `siteInfo` to the test environment. I don't really know how to do that. I tried searching code for similar use cases but couldn't find anything. Can you please help me with it :)